src/factories/ReportingTokenFactory.sol

```diff
@@ -2,10 +2,11 @@ pragma solidity ^0.4.13;
 
 import 'ROOT/libraries/Delegator.sol';
 import 'ROOT/reporting/ReportingToken.sol';
+import 'ROOT/reporting/Market.sol';
 
 
 contract ReportingTokenFactory {
-    function createReportingToken(Controller _controller, IMarket _market, int256[] _payoutNumerators) returns (ReportingToken) {
+    function createReportingToken(Controller _controller, Market _market, uint256[] _payoutNumerators) returns (ReportingToken) {
```

Comment:
Also ShareTokenFactory.createShareToken(), Branch.getReportingWindowForForkEndTime(), and 3 of the functions in ReportingToken.sol.